app/controllers/GoodsRecordsController.scala

```diff
@@ -48,36 +47,48 @@ class GoodsRecordsController @Inject() (
     extends FrontendBaseController
     with I18nSupport {
 
-  private val form = formProvider()
+  private val form     = formProvider()
+  private val pageSize = 10
```

Comment:
Could maybe make this a app config variable and then it might be easier to test pagination etc in future. Not big deal though.

Reply:
it doesn't change between environments so I'll keep it as it is for the moment